app/src/main/java/com/keylesspalace/tusky/components/accountlist/AccountListFragment.kt

```diff
@@ -330,6 +331,9 @@ class AccountListFragment :
                 val linkHeader = response.headers()["Link"]
                 onFetchAccountsSuccess(accountList, linkHeader)
             } catch (exception: Exception) {
+                if (exception is CancellationException) {
```

Comment:
could we add a comment? I don't understand where it will get caught and I wouldn't know why this catch is there in a week